http/headers/Report-To.json

```diff
@@ -10,7 +10,14 @@
             "chrome_android": "mirror",
             "edge": "mirror",
             "firefox": {
-              "version_added": false
+              "version_added": "130",
```

Comment:
Added in https://bugzilla.mozilla.org/show_bug.cgi?id=1860588 (stated in source)